.github/workflows/conformance-clustermesh.yaml

```
@@ -0,0 +1,503 @@
+
+    - name: Create Kind cluster 2
+      uses: helm/kind-action@d8ccf8fb623ce1bb360ae2f45f323d9d5c5e9f00
+      with:
+        cluster_name: ${{ env.clusterName2 }}
+        version: ${{ env.kind_version }}
+        config: ./.github/kind-config-cluster2.yaml
+        wait: 0 # The control-plane never becomes ready, since no CNI is present
+
+    - name: Wait for images to be available
+      timeout-minutes: 10
+      shell: bash
+      run: |
+        for image in cilium-ci operator-generic-ci hubble-relay-ci clustermesh-apiserver-ci ; do
+          until docker manifest inspect quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/$image:${{ steps.vars.outputs.sha }} &> /dev/null; do sleep 45s; done
+        done
+
+    - name: Install Cilium in cluster1
+      run: |
+        # Using the deprecated flag --cluster-name due to cilium/cilium-cli#1347
+        # --helm-set cluster.name ${{ env.clusterName1 }}
+        cilium --context ${{ env.contextName1 }} install \
+          ${{ steps.vars.outputs.cilium_install_defaults }} \
+          --encryption ${{ matrix.encryption }} \
+          --cluster-name ${{ env.clusterName1 }} \
+          --helm-set cluster.id=1
+
+    - name: Copy the IPsec secret to cluster2, as they must match
+      if: matrix.encryption == 'ipsec'
+      run: |
+        kubectl --context ${{ env.contextName1 }} get secret -n kube-system cilium-ipsec-keys -o yaml |
+          kubectl --context ${{ env.contextName2 }} create -f -
+
+    - name: Install Cilium in cluster2
+      run: |
+        # Using the deprecated form --cluster-name due to cilium/cilium-cli#1347
+        # --helm-set cluster.name ${{ env.clusterName2 }}
+        cilium --context ${{ env.contextName2 }} install \
+          ${{ steps.vars.outputs.cilium_install_defaults }} \
+          --encryption ${{ matrix.encryption }} \
+          --cluster-name ${{ env.clusterName2 }} \
+          --helm-set cluster.id=255 \
+          --inherit-ca ${{ env.contextName1 }}
+
+    - name: Enable Hubble
+      run: |
+        cilium hubble enable --context ${{ env.contextName1 }} ${{ steps.vars.outputs.hubble_enable_defaults }} --wait=false
+        cilium hubble enable --context ${{ env.contextName2 }} ${{ steps.vars.outputs.hubble_enable_defaults }} --relay=false
+        cilium status --wait --context ${{ env.contextName1 }}
+
+    - name: Enable Cluster Mesh
+      run: |
+        cilium --context ${{ env.contextName1 }} clustermesh enable ${{ steps.vars.outputs.clustermesh_enable_defaults }}
+        cilium --context ${{ env.contextName2 }} clustermesh enable ${{ steps.vars.outputs.clustermesh_enable_defaults }}
+
+    - name: Wait for cluster mesh status to be ready
+      run: |
+        cilium status --wait --context ${{ env.contextName1 }}
+        cilium status --wait --context ${{ env.contextName2 }}
+        cilium --context ${{ env.contextName1 }} clustermesh status --wait
+        cilium --context ${{ env.contextName2 }} clustermesh status --wait
```

nit: just to align with previous commands

```suggestion
        cilium clustermesh status --wait --context ${{ env.contextName1 }}
        cilium clustermesh status --wait --context ${{ env.contextName2 }}
 ```

---

I've moved the context to be the first parameter of all commands, for consistency